samples/web/content/apprtc/js/appwindow.js

```diff
@@ -61,10 +61,12 @@ var params = {
 // Generate random room id and connect.
 params.roomId = randomString(9);
 params.roomLink =  'https://apprtc.appspot.com/room/' + params.roomId;
-params.roomServer = 'https://apprtc.appspot.com';
+params.server = 'https://apprtc.appspot.com';
```

Comment:
Oops, that was a mistake of merging based on your old commit.


On Wed, Jan 14, 2015 at 10:01 AM, Chuck Hays <notifications@github.com>
wrote:

> In samples/web/content/apprtc/js/appwindow.js
> <https://github.com/GoogleChrome/webrtc/pull/374#discussion-diff-22955566>
> :
>
> > @@ -61,10 +61,12 @@ var params = {
> >  // Generate random room id and connect.
> >  params.roomId = randomString(9);
> >  params.roomLink =  'https://apprtc.appspot.com/room/' + params.roomId;
> > -params.roomServer = 'https://apprtc.appspot.com';
> > +params.server = 'https://apprtc.appspot.com';
>
> Had to line up with this change from the refactoring:
> +var Call = function(params) {
>
>    - this.params_ = params;
>    - this.roomServer_ = params.server || '';
>
> I could change it to roomServer in the other place if you prefer?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/webrtc/pull/374/files#r22955566>.
>



-- 
Jiayang Liu